packages/admin-next/dashboard/src/components/common/customer-info/customer-info.tsx

```diff
@@ -1,16 +1,16 @@
-import { Address, Cart, Order } from "@medusajs/medusa"
 import { Avatar, Copy, Text } from "@medusajs/ui"
 import { useTranslation } from "react-i18next"
 import { Link } from "react-router-dom"
+import { HttpTypes } from "@medusajs/types"
 import { getFormattedAddress, isSameAddress } from "../../../lib/addresses"
 
-const ID = ({ data }: { data: Cart | Order }) => {
+const ID = ({ data }: { data: HttpTypes.AdminOrder }) => {
   const { t } = useTranslation()
 
   const id = data.customer_id
-  const name = getCartOrOrderCustomer(data)
+  const name = getOrderCustomer(data)
   const email = data.email
-  const fallback = (name || email).charAt(0).toUpperCase()
+  const fallback = (name ?? email ?? "").charAt(0).toUpperCase()
```

Comment:
Q: is there a specific reason to go from `||` to `??`, I am asking being the behavior is different and it could lead to things like this:
```
"" ?? "my-email" -> ""
"" || "my-email" -> "my-email"
```

Reply:
In practise, you can only have `""` as name, if you've actively set it yourself. But I see your point, and we can change it back 👍

Reply:
so in that case the fallback would be `""` is it what we want?

Reply:
Changed it :)